### DISCUSSION TITLE
Welcome to Spotify-Connect Discussions!

### DISCUSSION THREAD

**Discussion by crnvl** [crnvl/Spotify-Connect#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajaske** [comment]
My name is Andy - a musician and retired economist/data analyst. I want to build a database of Spotify metrics for top songs. I can do it manually but it is a pain. Oh for some ideas on how to code that (or some help). I'm attaching a screenshot of the beginning of a file.[
![database start](https://user-images.githubusercontent.com/64706082/128900892-1703051d-1547-4870-945d-53362999b78d.png)
](url)
